packages/ui/src/council/modals/CandidacyPreview/CandidacyPreview.tsx

```diff
@@ -72,6 +73,13 @@ export const CandidacyPreview = React.memo(() => {
           />
         </SidePaneTopButtonsGroup>
       }
+      footer={
+        <ModalFooter>
+          <ButtonsGroup>
+            <ButtonPrimary size="large">Vote</ButtonPrimary>
+          </ButtonsGroup>
+        </ModalFooter>
+      }
     >
```

Comment:
```
footer={
  <ButtonsGroup>
    <ButtonPrimary size="small">Vote</ButtonPrimary>
  </ButtonsGroup>
}
```

There is no space for large button. Better to use 'small' for now and later I'll take a look at the design for this part.